Assets/Scripts/Player/PlayerController.cs

```diff
@@ -708,7 +712,7 @@ public class PlayerController : Entity {
 		anim.SetBool("JustFlipped", false);
 	}
 
-	IEnumerator RunDashCooldown(float seconds) {
+	public IEnumerator RunDashCooldown(float seconds) {
```

Comment:
I also don't see this referenced anywhere outside of PlayerController, I think it's safe to make it non-public.

Reply:
Yeah that was from a previous idea of how to implement the slide buff.